chainstate/src/detail/mod.rs

```diff
@@ -217,6 +219,38 @@ impl<S: BlockchainStorage, V: TransactionVerificationStrategy> Chainstate<S, V>
             config_geneis_id == stored_genesis_id,
             InitializationError::GenesisMismatch(config_geneis_id, stored_genesis_id),
         );
+
+        Ok(())
+    }
+
+    /// Check that transaction index state is consistent between DB and config.
+    fn process_tx_indexing(&mut self) -> Result<(), BlockError> {
```

Comment:
process_tx_index_enabled_flag, I would call this... does anyone have a better name?

Reply:
Sure, renamed.